tests/e2e/tasks/tenant/run_workload.yml

```diff
@@ -24,6 +24,13 @@
         replace: "boot_disk_copy: true"
       when: set_boot_disk_copy|default(false)|bool
 
+    - name: set data copy
+      ansible.builtin.replace:
+        path: "{{ os_migrate_data_dir }}/workloads.yml"
+        regexp: "data_copy: true"
+        replace: "data_copy: false"
+      when: set_no_data_copy|default(true)|bool
```

Comment:
I think this behaves inverse to what was intended and what the comments in workload migration tests say. We should default to not doing anything here (to keep existing e2e workload tests working the same way).

So i think we should have:

```
when: set_no_data_copy|default(false)|bool
```